### PR TITLE
Improve user search dropdown

### PR DESCRIPTION
**Pull request by annatraussnig**
Add user name and portrait to the dropdown results (NB: add the `element` param for `Autocomplete` to styleguide docs - atm only `text` is mentionned, I believe)

### CODE REVIEW

File: components/editor/modules/link/ui.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Text } from 'slate'
 import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js:3]: is this necessary? auto add by IDE?

File: components/editor/modules/link/ui.js [hunk 2]
```diff
@@ -19,17 +20,37 @@ const getUsers = gql`
...
   }
 `
 
+const UserItem = ({ user }) => (
+  <div style={{ display: 'flex', alignItems: 'center' }}>
+    <img
+      src={user.portrait}
+      width='100%'
+      style={{ width: 54, marginRight: 10 }}
+    />
```
Comment (tpreusse) [inline components/editor/modules/link/ui.js:34]: I'd this align with [the frontend profile](https://github.com/orbiting/republik-frontend/blob/2753d6a152ed7499c81642b2a33b743d25b8e955/components/Profile/Portrait.js#L10-L17) and display an gray area when there is none.
```suggestion
    <div
      style={{
        width: 54,
        height: 54,
        backgroundColor: '#E2E8E6',
        backgroundImage: user.portrait ? `url(${user.portrait})` : undefined,
        backgroundSize: 'cover',
        backgroundPosition: 'center',
        marginRight: 10
      }}
    />
```

**Review by tpreusse — COMMENTED** [review]
Nice, one suggestion re profile pic

File: components/editor/modules/link/ui.js (hunk 1 above)
Comment (annatraussnig) [reply components/editor/modules/link/ui.js:3]: leftover 😬

**Review by tpreusse — APPROVED** [review]
lgtm

**Pull request merged**